docs/rpc-endpoints.md

```diff
@@ -73,6 +73,72 @@ Possible values for the "reason" field and "reason_data" field are:
 Reason types without additional information will not have a
 `reason_data` field.
 
+### GET /v2/pox
+
+Get current PoX-relevant information.
+
+Returns JSON data in the form:
+
+```
+{
+  "contract_id": "SP000000000000000000002Q6VF78.pox",
+  "first_burnchain_block_height": 666050,
+  "next_cycle_cur_threshold": 70000000000,
+  "cur_cycle_threshold": 70000000000,
+  "cur_cycle_stacked_ustx": 202157971547640,
+  "next_cycle_stacked_ustx": 151807880982060,
+  "reward_slots": 4000,
+  "next_rewards_start": 672350,
+  "next_prepare_phase_start": 672250,
+  "min_stacking_increment_ustx": 52251700044,
+  "pox_activation_threshold": 52251700044388,
+  "prepare_cycle_length": 100,
+  "rejection_fraction": 25,
+  "reward_cycle_id": 2,
+  "reward_cycle_length": 2100,
+  "rejection_votes_left_required": 261258500221925,
+  "total_liquid_supply_ustx": 1045034000887772,
+  "next_reward_cycle_in": 1318,
+  "next_prepare_phase_in": 1218
+}
+```
+
+* `contract_id` is the contract identifier for the PoX contract.
+* `cur_cycle_threshold` is the threshold amount for obtaining a slot in the
+  active reward cycle.
+* `next_cycle_cur_threshold` is the _current_ microstacks threshold for the
+  _next_ reward cycle. If more STX is locked for participation in that cycle,
+  the threshold may increase.
+* `first_burnchain_block_height` is the first burn block evaluated in this Stacks
+  chain.
+* `cur_cycle_stacked_ustx` is the total amount of stacked microstacks in the
+  active reward cycle.
+* `next_cycle_stacked_ustx` is the total amount of stacked microstacks in the
+  next reward cycle.
+* `reward_slots` is the number of reward slots in a reward cycle
+* `next_rewards_start` is the burn block height when the next reward
+  cycle begins
+* `next_prepare_phase_start` is the burn block height when the next prepare
+  phase begins. Any eligible stacks _must_ be stacked before this block.
+* `min_stacking_increment_ustx` is the minimum amount that can be used to
```

Comment:
I guess I'm a little bit confused by this value.  Is this the number of uSTX required to clinch a single reward slot?  Because if so, shouldn't it always be a multiple of 10,000 STX?

Reply:
It's the amount that will result in an immediate rejection from `stack-stx`

Reply:
Ah, makes sense :+1: 

Should there be a field in /v2/pox that tells you how many uSTX you need to lock to get a reward slot?  I have a script [here](https://github.com/jcnelson/get-stacking-info.sh/blob/main/get-stacking-info.sh) that I use for this, but I could easily see this information getting added to this endpoint.

Reply:
That's _sort of_ what `next_cycle_cur_threshold` and `cur_cycle_threshold` tell you -- `cur_cycle_threshold` tells you the amount that would have needed to be stacked to get 1 reward address in the current cycle, `next_cycle_cur_threshold` tells you the amount that would be needed for the next cycle, assuming no more participation occurs.

Reply:
Ah, okay. that's what I was looking for. Thanks!